### PR TITLE
Docs 4911 Updates

### PR DESCRIPTION
**Pull request by valkyrie69**
Bundled a couple of changes in this PR.

Note about 100 bg limit per org
Reorganized content so Business Groups has it's own page and is more prominent in the TOC
Edits
Explanation of Anypoint Staff entry in audit logs

### CODE REVIEW

File: modules/ROOT/pages/audit-logging.adoc [hunk 1]
```diff
@@ -11,7 +11,9 @@ Audit logs have a retention period of six years. Download your logs periodically
 
 == Access Audit Logging
 
-Users belonging to the *Organization Administration* role or the *Audit Log Viewer* role on the Anypoint Platform have access to both the UI and the Query API. The audit log service is business group aware, meaning, you will see only logs that are relevant to your own business group. The audit log UI is embedded in Access Manager. If you are not an *Organization Administrator*, but you have the *Audit Log Viewer* role, then you can see only the audit log user interface under Access Manager.
+Users belonging to the *Organization Administration* role or the *Audit Log Viewer* role on the Anypoint Platform have access to both the UI and the Query API. The audit log service is business group aware, which means you see only logs that are relevant to your own business group.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/audit-logging.adoc]: no "the" before Anypoint Platform

Comment (Mulesoft-Mysti) [reply modules/ROOT/pages/audit-logging.adoc]: And should that be business-group aware, since "business group" modifies "aware"?

File: modules/ROOT/pages/audit-logging.adoc [hunk 2]
```diff
@@ -11,7 +11,9 @@ Audit logs have a retention period of six years. Download your logs periodically
 
 == Access Audit Logging
 
-Users belonging to the *Organization Administration* role or the *Audit Log Viewer* role on the Anypoint Platform have access to both the UI and the Query API. The audit log service is business group aware, meaning, you will see only logs that are relevant to your own business group. The audit log UI is embedded in Access Manager. If you are not an *Organization Administrator*, but you have the *Audit Log Viewer* role, then you can see only the audit log user interface under Access Manager.
+Users belonging to the *Organization Administration* role or the *Audit Log Viewer* role on the Anypoint Platform have access to both the UI and the Query API. The audit log service is business group aware, which means you see only logs that are relevant to your own business group. 
+
+The audit log UI is embedded in Access Manager. If you are not an *Organization Administrator*, but you have the *Audit Log Viewer* role, you can see only the audit log user interface under Access Manager.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/audit-logging.adoc]: be consistent UI or user interface (I think UI has won the battle in our doc?)

File: modules/ROOT/pages/audit-logging.adoc [hunk 3]
```diff
@@ -29,7 +31,18 @@ Each log entry has a set of properties that provides information about the activ
 * *Type*: The type of the object on which the action is performed, for example, `Organization`
 * *Action*: The action associated with the object, for example, `Create`
 * *Object*: The name of the Object, for example, `foo`
-* *User*: The User who performed the action, for example, `johndoe`
+* *User*: The User who performed the action, for example, `johndoe` 
++
+[NOTE]
+--
+You may occasionally see an *Anypoint Staff* log entry. This log entry is made when the audit action was performed by a user who does not belong to your master organization. +
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/audit-logging.adoc]: may is a translation problem, how about Some log entries are marked as `Anypoint Staff` (this is a log entry, not a UI label, yes?).

File: modules/ROOT/pages/audit-logging.adoc [hunk 4]
```diff
@@ -29,7 +31,18 @@ Each log entry has a set of properties that provides information about the activ
...
+These actions include when:
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/audit-logging.adoc]: I don't think you should separate the object of the preposition (the stuff after "when") across a list & intro. How about `Anypoint Staff` entries are created for when these events occur:
* An internal or system process performs a task
* A MuleSoft Sales staff member changes entitlements for your organization
* A MuleSoft employee takes an action using a bearer token...

File: modules/ROOT/pages/business-groups.adoc [hunk 5]
```diff
@@ -0,0 +1,102 @@
+= Business Groups
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Business groups are also referred to as organizations (or sub organizations). Business groups reside within the master organization and can contain their own business groups. The master organization and its business groups are organized in a hierarchical tree, where the master organization is the root. Each business group has one direct parent and can have multiple children.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/business-groups.adoc]: I wouldn't lead with the most confusing thing, that we use two different terms for the same thing and one of those terms also means something else. maybe delete or move the first sentence?

Comment (Mulesoft-Mysti) [reply modules/ROOT/pages/business-groups.adoc]: I read that whole paragraph and I still don't know what business groups do, what user problem or goal they satisfy.

Comment (Mulesoft-Mysti) [reply modules/ROOT/pages/business-groups.adoc]: Is the master organization  also a parent business group (the highest biz group?). I think so but am not sure...

Comment (Mulesoft-Mysti) [reply modules/ROOT/pages/business-groups.adoc]: Also, global comment, it must be either sub-organization or suborganization, not sub organization.

File: modules/ROOT/pages/business-groups.adoc [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+
+[NOTE]
+Business groups are not enabled by default in a new Anypoint Platform account. To activate business groups in your organization, contact your MuleSoft representative. There is a limit of 100 business groups per organization.
+
+If you are an organization administrator and your organization is enabled for business group support, you can create business groups within your organization. The organization administrator is also the administrator of any business groups (sub organizations) within the parent organization.
+
+[NOTE]
+When you create a new business group, all the users with the *Organization Administrators* role in the parent business group are also added as organization administrators in the newly created business group (the child business group). However, when you create a new user as an *Organization Administrator* in the parent business group, the user is not added to all the sub business groups.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/business-groups.adoc]: We should probably use either sub-org or child org, but not both.

File: modules/ROOT/pages/business-groups.adoc [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+
+=== Create a Business Group
+
+Create business groups within your organization. 
+
+[NOTE]
+There is a limit of 100 business groups per organization.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/business-groups.adoc]: Instead of burying this limit on the task topic in a note, maybe add to conceptual topic and give it a heading that includes Limit?

File: modules/ROOT/pages/business-groups.adoc [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+. Click the blue plus sign next to the organization name or any of the business groups in the tree diagram.
++
+image::organization-20885.png[]
++
+. In the dialog box, enter:
+  * *Business Group name*: Name for the new business group.
+  * *Owner*: Assign an existing user in the organization to be the business group owner.
+  * Select *Owner can create Business Groups* to allow the owner to create business groups that branch under this business group.
+  * Select *Owner can create environments* to allow the owner to create environments within this business group.
+  * You can assign some or all of the vCores that your organization owns to an individual business group. This ensures that these are used by the CloudHub deployments within the business group. You can assign these when creating the business group or edit these settings later.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/business-groups.adoc]: how is a deployment in a business group? We deploy runtimes and apps ....?

Comment (Mulesoft-Mysti) [reply modules/ROOT/pages/business-groups.adoc]: also "these" it isn't clear what "these" are. vCores?

File: modules/ROOT/pages/business-groups.adoc [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+. Click *Add Business Group*.
+
+image::organization-9fa5f.png[]
+
+[NOTE]
+--
+Allocating vCores to a business group makes those vCores available only to that business group, which makes them unavailable to the parent organization.
+--
+
+==== Child Business Groups
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/business-groups.adoc]: If this is a conceptual section, should it be in the first section, before any tasks?

File: modules/ROOT/pages/business-groups.adoc [hunk 10]
```diff
@@ -0,0 +1,102 @@
...
+
+You can create hierarchical levels of business groups to provide more control over access. To create a child business group click the blue *+* icon next to any business group.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/business-groups.adoc]: This task, even though short, should probably have its own heading, and be among the tasks, not here in the middle of a conceptual section?

File: modules/ROOT/pages/business-groups.adoc [hunk 11]
```diff
@@ -0,0 +1,102 @@
...
+
+=== Navigating Between Business Groups
+
+When your organization has multiple business groups, you can navigate between them using the menu at the top-right corner. Switching between business groups changes the list of available CloudHub deployments, APIs, users, and roles settings.
+
+image::switch-suborg.png[switch+suborg]
+
+[NOTE]
+--
+When you sign into Anypoint Platform, you are signed back into the organization or business group you were in when you last signed out.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/business-groups.adoc]: how does that work? If I am a member of a biz group, am I always signed back into it? is the master org also a "master" biz group if that's enabled, even if I haven't created any biz groups yet?

File: modules/ROOT/pages/business-groups.adoc [hunk 12]
```diff
@@ -0,0 +1,102 @@
...
+
+Roles can be applied at the master organization level as well as at the business group level. However, each controls different resources. Roles and permissions assigned to a user in one particular business group (or the master org) are independent of the roles and permissions assigned to that same user in other business groups.
+
+To obtain the membership for a business group, you must be granted a role within that business group. Once you are visible in a business group, you can be granted individual permissions in that business group. All users, regardless of which business groups they are in, belong to the master organization and can be assigned individual permissions in the master organization.
+
+Members that are added to a business group can view and access the business group, but to access certain resources, you must have the appropriate roles assigned to you. For example, to access APIs and CloudHub deployments in the business group, you must be assigned roles that belong to that business group. Additionally, roles that belong to a business group can only grant access to APIs and CloudHub deployments within that same business group.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/business-groups.adoc]: how are APIs or deployments (!) associated with a business group? "in" is confusing me (just me?)

Comment (Mulesoft-Mysti) [reply modules/ROOT/pages/business-groups.adoc]: you only need to tell me this in one place (first section), and if it's there and I didn't get it, my apologies. I understand how users & roles are associated with a biz group, but not other objects like APIs or *deployments*?! Is the runtime engine associated? apps in the runtime engine?

File: modules/ROOT/pages/organization.adoc [hunk 13]
```diff
@@ -4,28 +4,24 @@ include::_attributes.adoc[]
...
 
-The organization is an account where multiple users can share resources, including applications and environments. The level of access users have to various resources depends on their assigned roles and permissions. For example, one user might have permission to manage API alerts, while another user has permission only to view API alerts.
+The organization name is the name you entered in the *Company* field in the initial Anypoint Platform signup form. You can change the name in the organization settings.
 
-When you sign into Anypoint Platform, the organization icon and name are displayed on the top right of the page. You can click the organization icon to navigate between business groups. The master organization can contain multiple business groups. You can think of business groups as sub organizations, or children of the master organization.
+An organization is an account where multiple users can share resources, including applications and environments. The level of access users have to various resources depends on their assigned roles and permissions. For example, one user might have permission to manage API alerts, while another user has permission only to view API alerts.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/organization.adoc:11]: master org = account, yes? Can we use just one term?

File: modules/ROOT/pages/organization.adoc [hunk 14]
```diff
@@ -4,28 +4,24 @@ include::_attributes.adoc[]
...
+
+When you sign into Anypoint Platform, the organization icon and name are displayed on the top right of the page. The master organization can contain multiple business groups. You can think of business groups as sub organizations, or children of the master organization. Click the organization icon to navigate between business groups.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/organization.adoc]: sub-org or suborg, not sub org, which would be an org of submarines...

File: modules/ROOT/pages/organization.adoc [hunk 15]
```diff
@@ -4,28 +4,24 @@ include::_attributes.adoc[]
...
+An organization is an account where multiple users can share resources, including applications and environments. The level of access users have to various resources depends on their assigned roles and permissions. For example, one user might have permission to manage API alerts, while another user has permission only to view API alerts.
+
+When you sign into Anypoint Platform, the organization icon and name are displayed on the top right of the page. The master organization can contain multiple business groups. You can think of business groups as sub organizations, or children of the master organization. Click the organization icon to navigate between business groups. 
 
 [NOTE]
---
-When you sign into Anypoint Platform, you are signed back into whichever organization, or business group, you were in when you last signed out.
---
+When you sign into Anypoint Platform, you are signed back into the organization or business group you were in when you last signed out. When you are working, always verify you are in the correct organization or business group.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/organization.adoc]: You can delete "When you are working"

File: modules/ROOT/pages/business-groups.adoc (hunk 11 above)
Comment (Mulesoft-Mysti) [reply modules/ROOT/pages/business-groups.adoc]: Also, I like the caveat you include with this in organizations.adoc: "always verify you are in the correct organization or business group."

File: modules/ROOT/pages/organization.adoc [hunk 16]
```diff
@@ -35,44 +31,44 @@ An Anypoint Platform user who has been granted organization administrator privil
...
 +
-The *Organization* page is displayed, where you can view organizations and business groups and add new business groups (if enabled). +
+The *Organization* page is displayed, where you can view organizations and business groups and add new business groups (if enabled). 
+
 You can also click *Organization* in the left menu to view the Organization page.
 
 === Client ID and Client Secret
 
-Each master organization, environment, and business group within the master organization has its own associated unique client ID and client secret. These are used for authentication by users who are not organization administrators to access assets within an organization. The client ID and password are generated by Anypoint Platform for each environment that you create, and they are globally unique.
+Each master organization, environment, and business group within the master organization has its own associated unique client ID and client secret. These are used for authentication by users who are not organization administrators to access assets within an organization. The client ID and password are generated by Anypoint Platform for each environment you create, and they are globally unique.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/organization.adoc:57]: If Organizaiton Administrator is capitalized everywhere else, cap it here?

File: modules/ROOT/pages/organization.adoc [hunk 17]
```diff
@@ -35,44 +31,44 @@ An Anypoint Platform user who has been granted organization administrator privil
 * Access analytics for the APIs in your organization
 * Create business groups to delegate management of the resources and define the scopes of roles and permissions. A business group owner can configure additional properties at the business group level.
 
+When the organization owner creates a business group, an owner must be assigned to it. This assigned user holds an *Organization Administrators* role within that business group by default.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/organization.adoc]: but since the creator is automatically given this role, are we saying a 2nd owner must be assgined? Or "owner" has some meaning outside of Organization Administrators role?

**Review by Mulesoft-Mysti — APPROVED** [review]
Love the improvements! No show-stopper comments.

File: modules/ROOT/pages/organization.adoc (hunk 13 above)
Comment (valkyrie69) [reply modules/ROOT/pages/organization.adoc:11]: Terminology needs to be straightened out for Access Management for sure. But it's going to exceed the purpose of this PR and input from the PM. For now I'm not tackling it.

File: modules/ROOT/pages/organization.adoc (hunk 14 above)
Comment (valkyrie69) [reply modules/ROOT/pages/organization.adoc]: lol

File: modules/ROOT/pages/organization.adoc (hunk 16 above)
Comment (valkyrie69) [reply modules/ROOT/pages/organization.adoc:57]: This is another thing that is inconsistent, but I was trying to capitalize only when referring directly to the role, then not capitalizing when it's the concept of an organization administrator.

File: modules/ROOT/pages/organization.adoc (hunk 17 above)
Comment (valkyrie69) [reply modules/ROOT/pages/organization.adoc]: Owner is not a role. Every business group has to have an owner and every owner is automatically assigned the role of Organization Administrator.

File: modules/ROOT/pages/audit-logging.adoc [hunk 18]
```diff
@@ -11,7 +11,9 @@ Audit logs have a retention period of six years. Download your logs periodically
 
 == Access Audit Logging
 
-Users belonging to the *Organization Administration* role or the *Audit Log Viewer* role on the Anypoint Platform have access to both the UI and the Query API. The audit log service is business group aware, meaning, you will see only logs that are relevant to your own business group. The audit log UI is embedded in Access Manager. If you are not an *Organization Administrator*, but you have the *Audit Log Viewer* role, then you can see only the audit log user interface under Access Manager.
+Users belonging to the *Organization Administration* role or the *Audit Log Viewer* role on Anypoint Platform have access to both the UI and the Query API. The audit log service is business-group aware, which means you see only logs that are relevant to your own business group. 
+
+The audit log UI is embedded in Access Manager. If you are not an *Organization Administrator*, but you have the *Audit Log Viewer* role, you can see only the audit log UI under Access Manager.
```
Comment (sebastiankorol) [inline modules/ROOT/pages/audit-logging.adoc]: nit: Access Management

File: modules/ROOT/pages/audit-logging.adoc [hunk 19]
```diff
@@ -29,7 +31,18 @@ Each log entry has a set of properties that provides information about the activ
 * *Type*: The type of the object on which the action is performed, for example, `Organization`
 * *Action*: The action associated with the object, for example, `Create`
 * *Object*: The name of the Object, for example, `foo`
-* *User*: The User who performed the action, for example, `johndoe`
+* *User*: The User who performed the action, for example, `johndoe` 
++
+[NOTE]
+--
+Occasionally, there are `Anypoint Staff` log entries. This log entry is made when the audit action was performed by a user who does not belong to your master organization. +
```
Comment (sebastiankorol) [inline modules/ROOT/pages/audit-logging.adoc]: I think "master" organization can be confusing here. Since all users belong to the master org. Can we just say "your organization"?

File: modules/ROOT/pages/audit-logging.adoc [hunk 20]
```diff
@@ -29,7 +31,18 @@ Each log entry has a set of properties that provides information about the activ
...
+`Anypoint Staff` entries are created for when these events occur:
+
+* An internal or system processes is performed
+* The SalesOps staff changes entitlements for your organization
```
Comment (sebastiankorol) [inline modules/ROOT/pages/audit-logging.adoc]: SalesOps -> Mulesoft Procurement as a result of license grants or similar

File: modules/ROOT/pages/audit-logging.adoc [hunk 21]
```diff
@@ -29,7 +31,18 @@ Each log entry has a set of properties that provides information about the activ
...
+* *User*: The User who performed the action, for example, `johndoe` 
++
+[NOTE]
+--
+Occasionally, there are `Anypoint Staff` log entries. This log entry is made when the audit action was performed by a user who does not belong to your master organization. +
+`Anypoint Staff` entries are created for when these events occur:
+
+* An internal or system processes is performed
+* The SalesOps staff changes entitlements for your organization 
+* An action is taken using a bearer token associated with a MuleSoft employee account (for example, during troubleshooting)
```
Comment (sebastiankorol) [inline modules/ROOT/pages/audit-logging.adoc]: I know we got this directly from support, but the way it's worded is very much tied to how support thinks and troubleshoots the platform. Bearer tokens here don't really mean much. We should say something like "An action is taken by a MuleSoft employee on your behalf for other purposes, such as troubleshooting". Note the "other", since it's really a less common variation of the above bullet

File: modules/ROOT/pages/audit-logging.adoc [hunk 22]
```diff
@@ -29,7 +31,18 @@ Each log entry has a set of properties that provides information about the activ
...
 * *Object*: The name of the Object, for example, `foo`
-* *User*: The User who performed the action, for example, `johndoe`
+* *User*: The User who performed the action, for example, `johndoe` 
++
+[NOTE]
+--
+Occasionally, there are `Anypoint Staff` log entries. This log entry is made when the audit action was performed by a user who does not belong to your master organization. +
+`Anypoint Staff` entries are created for when these events occur:
+
+* An internal or system processes is performed
```
Comment (sebastiankorol) [inline modules/ROOT/pages/audit-logging.adoc]: nit: process.

We could also clarify that these are part of the normal anypoint platform operation. Not distinct, meaningful actions

File: modules/ROOT/pages/business-groups.adoc [hunk 23]
```diff
@@ -0,0 +1,116 @@
+= Business Groups
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Business groups provide a way to separate and control access to Anypoint Platform resources such as APIs and applications. Users have access only to the business groups to which they are added.  
+
+Business groups reside within the master organization and can contain their own child business groups. The master organization and its business groups are organized in a hierarchical tree, where the master organization is the root. Each business group has one direct parent and can have multiple children.
+
+[NOTE]
```
Comment (sebastiankorol) [inline modules/ROOT/pages/business-groups.adoc:10]: 100 BG limit is duplicated from the other note below

File: modules/ROOT/pages/business-groups.adoc [hunk 24]
```diff
@@ -0,0 +1,116 @@
+= Business Groups
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Business groups provide a way to separate and control access to Anypoint Platform resources such as APIs and applications. Users have access only to the business groups to which they are added.
```
Comment (sebastiankorol) [inline modules/ROOT/pages/business-groups.adoc]: I think "Users have access only to the business groups to which they are added" is slightly misleading because the question that follows is "how do I add a user to a business group", and that makes users think of invites. Can we instead say that users are only members of Business Groups (and therefore show up in the user list) iff they have a role in that BG?

File: modules/ROOT/pages/business-groups.adoc [hunk 25]
```diff
@@ -0,0 +1,116 @@
...
+Business groups are not enabled by default in a new Anypoint Platform account. To activate business groups in your organization, contact your MuleSoft representative. There is a limit of 100 business groups per organization.
+
+If you are an organization administrator and your organization is enabled for business group support, you can create business groups within your organization. The organization administrator is also the administrator of any business groups (sub-organizations) within the parent organization.
+
+[NOTE]
+There is a limit of 100 business groups per organization. 
+
+==== Child Business Groups
+
+Child business groups provide more fine grained control over access.
```
Comment (sebastiankorol) [inline modules/ROOT/pages/business-groups.adoc]: nit: over access to resources

File: modules/ROOT/pages/business-groups.adoc [hunk 26]
```diff
@@ -0,0 +1,116 @@
...
+[NOTE]
+There is a limit of 100 business groups per organization. 
+
+==== Child Business Groups
+
+Child business groups provide more fine grained control over access. 
+
+image::organization-2b25c.png[]
+
+Since business groups are hierarchical, the owner of a parent business group always retains administrator permissions for any child business group within that parent. This means that the owner of a business group can create child business groups and assign another user as the owner (owners automatically have the *Organization Administrators* role) of those child business groups. The owner of the parent group remains an *Organization Administrator* within the child business group.
```
Comment (sebastiankorol) [inline modules/ROOT/pages/business-groups.adoc]: _Always_ isn't true. They always retain the ability to make themselves owners and admins, but the hierarchy grants admin access at BG creation time. Therefore, if a user is added as an admin of the parent org later, they won't have automatic admin of the BG

File: modules/ROOT/pages/business-groups.adoc [hunk 27]
```diff
@@ -0,0 +1,116 @@
...
+image::organization-2b25c.png[]
+
+Since business groups are hierarchical, the owner of a parent business group always retains administrator permissions for any child business group within that parent. This means that the owner of a business group can create child business groups and assign another user as the owner (owners automatically have the *Organization Administrators* role) of those child business groups. The owner of the parent group remains an *Organization Administrator* within the child business group. 
+
+Conversely, owners of child business groups cannot:
+
+* Access or modify the parent business group or master organization settings
+* View the parent business group's client ID and client secret
+
+When you create a child business group within a parent, only the vCores that were assigned to the parent business group can be allocated to the child business group.
```
Comment (sebastiankorol) [inline modules/ROOT/pages/business-groups.adoc]: nit: vCores -> redistributable entitlements. VPCs and Load Balancers also fall in this category

File: modules/ROOT/pages/business-groups.adoc [hunk 28]
```diff
@@ -0,0 +1,116 @@
...
+
+=== Create a Business Group
+
+Create business groups within your organization. When you create a new business group, all the users with the *Organization Administrators* role in the parent business group are also added as organization administrators in the newly created business group. However, when you create a new user as an *Organization Administrator* in the parent business group, the user is not added to child business groups.
```
Comment (sebastiankorol) [inline modules/ROOT/pages/business-groups.adoc]: re: `the user is not added to child business groups.` -- I think it's extremely important that this section clarifies that you can't be "added" to a BG. You are always a member of the master org. Whether your user shows up in the list of users in the BG is dependent on roles and permissions, not "membership"

File: modules/ROOT/pages/business-groups.adoc (hunk 11 above)
Comment (sebastiankorol) [reply modules/ROOT/pages/business-groups.adoc]: Let's reword so that you're not "signed into" a BG, but rather have an active one. You're always signed into your user account, not an organization.

File: modules/ROOT/pages/business-groups.adoc [hunk 29]
```diff
@@ -0,0 +1,116 @@
...
+
+If you also have the *Organization Administrators* role in BusinessGroupB, this does not mean that you can grant access to APIs and CloudHub deployments in BusinessGroupA to users in BusinessGroupB and vice versa.
+
+Resources that belong to the master organization require roles specified at the master organization level.
+
+Users who are assigned the *Organization Administrators* role in the master organization, or any business group, are automatically assigned that same role in any child business group created within that business group, so are automatically visible in the child business group. This does not apply to users who are assigned the *Organization Administrators* role after a child business group has been created.
+
+=== Delete a Business Groups
+
+Only an organization administrator can delete business groups. The master (root) organization can't be deleted, even by an organization administrator.
```
Comment (sebastiankorol) [inline modules/ROOT/pages/business-groups.adoc]: nit: Why are we clarifying the meaning of master org here, but not elsewhere?

File: modules/ROOT/pages/organization.adoc (hunk 13 above)
Comment (sebastiankorol) [reply modules/ROOT/pages/organization.adoc:11]: "account" is overloaded between user account and tenant. I'd rather not use account at all if possible.

I agree with this being out of scope of the PR, but my personal perference would be something like:
BG: A node in a tree of (mostly) self-contained resource groups.
Organization: The whole hierarchy of BGs
Master or root BG: the top-level BG that can't be deleted

File: modules/ROOT/pages/organization.adoc [hunk 30]
```diff
@@ -4,28 +4,24 @@ include::_attributes.adoc[]
...
+An organization is an account where multiple users can share resources, including applications and environments. The level of access users have to various resources depends on their assigned roles and permissions. For example, one user might have permission to manage API alerts, while another user has permission only to view API alerts.
+
+When you sign into Anypoint Platform, the organization icon and name are displayed on the top right of the page. The master organization can contain multiple business groups. You can think of business groups as sub-organizations, or children of the master organization. Click the organization icon to navigate between business groups. 
 
 [NOTE]
---
-When you sign into Anypoint Platform, you are signed back into whichever organization, or business group, you were in when you last signed out.
---
+When you sign into Anypoint Platform, you are signed back into the organization or business group you were in when you last signed out. Always verify you are in the correct organization or business group.
```
Comment (sebastiankorol) [inline modules/ROOT/pages/organization.adoc]: same as above -- we don't sign into organizations/BGs. We sign into user accounts. Also, can we consolidate this note in just a single place?

File: modules/ROOT/pages/organization.adoc [hunk 31]
```diff
@@ -4,28 +4,24 @@ include::_attributes.adoc[]
...
-
-Other users can be assigned the Organization Administrators role, but there is only one organization owner per Anypoint Platform account (master organization). The owner can transfer ownership of the account to another user, but there cannot be more than one owner at once.
+The user who first signs up for an Anypoint Platform account is designated as the organization owner. This is not a role that can be assigned, rather it is an identifier for this single user, who is assigned the *Organization Administrators* role by default. 
 
-When the organization owner creates a business group, an owner must be assigned to it. This assigned user holds an Organization Administrators role within that business group by default.
+Other users can be assigned the *Organization Administrators* role, but there is only one organization owner per Anypoint Platform account (master organization). The owner can transfer ownership of the account to another user, but there cannot be more than one owner at once. The *Organization Administrators* role cannot be removed from the organization owner in the master organization or any business group that belongs to the master organization.
```
Comment (sebastiankorol) [inline modules/ROOT/pages/organization.adoc]: Any org admin can change owners, not just _the_ owner. The new owner must also be an org admin

File: modules/ROOT/pages/organization.adoc [hunk 32]
```diff
@@ -35,44 +31,44 @@ An Anypoint Platform user who has been granted organization administrator privil
...
+Each master organization, environment, and business group within the master organization has its own associated unique client ID and client secret. These are used for authentication by users who are not organization administrators to access assets within an organization. The client ID and password are generated by Anypoint Platform for each environment you create, and they are globally unique.
 
 To deploy proxies or APIs to CloudHub, you must use these values to configure a customer-hosted Mule Runtime or legacy API Gateway.
 
 [NOTE]
---
-Organization-level client IDs and client secrets are supported only for backward compatibility. In newer Anypoint Platform accounts, you must instead use the client ID and client secret for an _environment_. See xref:environments.adoc[Environments].
---
+Organization-level client IDs and client secrets are supported only for backward compatibility. In newer Anypoint Platform accounts, you must instead use the client ID and client secret for an environment. See xref:environments.adoc[Environments].
```
Comment (sebastiankorol) [inline modules/ROOT/pages/organization.adoc]: "Must" carries a strong meaning that we can't enforce. Prefer "should", here. But I believe this is out of scope for Access Management and should be documented in API Manager

**Review by sebastiankorol — CHANGES_REQUESTED** [review]
Thank you Valkyrie! Some very important additions here. I would definitely like to change the notion of "adding" users to BGs. Everything else is just nits.

File: modules/ROOT/pages/audit-logging.adoc (hunk 18 above)
Comment (valkyrie69) [reply modules/ROOT/pages/audit-logging.adoc]: Nits are good.

File: modules/ROOT/pages/business-groups.adoc [hunk 33]
```diff
@@ -0,0 +1,111 @@
+= Business Groups
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Business groups are self-contained resource groups that contain Anypoint Platform resources such as APIs and applications. Business groups provide a way to separate and control access to Anypoint Platform resources, as users have access only to the business groups in which they have a role. 
+
+Business groups reside within the master organization and are organized in a hierarchical tree where the top-level business group is the root. You can add up to 100 business groups per organization. Each business group you create has one direct parent and can have multiple children.
```
Comment (sebastiankorol) [inline modules/ROOT/pages/business-groups.adoc]: 100 BG limit is duplicated here and in the paragraph below. Can we remove it from here?

File: modules/ROOT/pages/business-groups.adoc [hunk 34]
```diff
@@ -0,0 +1,111 @@
...
+--
+
+=== Create a Child Business Group
+
+You can create hierarchical levels of business groups to provide more control over access. 
+
+. Click the blue *+* icon on the left of the business group in which to create the child. 
+. In the *Add Business Group* dialog, enter a *Business Group name* and *Owner*, then select from these options: +
+* *Owner can create Business Groups*: The owner you assign to the business group can create child business groups within the business group for which they are the owner. 
+* *Owner can create environments*: The owner you assign can create environments within the business group.
```
Comment (sebastiankorol) [inline modules/ROOT/pages/business-groups.adoc]: We should not use "owner" as an authorization mechanism. Let's reword to "organization administrators can create environments within the business group"

**Review by sebastiankorol — APPROVED** [review]
LGTM. Just two outstanding nits: 100 BG limit being duplicated and use of "owner" as a role
